SConstruct

```diff
@@ -1287,9 +1287,21 @@ if env["PLATFORM"] == "win32" :
 						os.path.dirname(filePath ),
 						sourceFile
 					) ) ):
-						fileOrigin[ os.path.join( env.subst( "$BUILD_DIR" ), filePath ) ]= os.path.abspath(
+						fileOrigin[ filePath ] = os.path.abspath(
 							os.path.join( os.path.dirname( filePath ), sourceFile ) 
-						)
+						).replace( os.getcwd(), "" ).lstrip( os.path.sep )
+
+	def customInstaller( dest, source, env ) :
+
+		source = fileOrigin.get( source, source )
```

Comment:
I was hoping we could do the "is it a symlink" check directly here, to avoid having to build the `fileOrigin` dict (which will include many files that we will never install).

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/4524/commits/e2b558ae77891bcb2b382064366fbe443a701349